app/Http/Controllers/API/v1/TransportController.php

```diff
@@ -72,4 +77,74 @@ public function getTrip(Request $request): JsonResponse {
 
         return $this->sendv1Response(data: $trainTripResponse);
     }
+
+
+    public function create(Request $request): JsonResponse {
+        $validator = Validator::make($request->all(), [
+            'body'        => 'max:280',
+            'business'    => Rule::in(Business::getList()),
+            'visibility'  => Rule::in(StatusVisibility::getList()),
+            'event'       => 'integer',
+            'tweet'       => 'boolean',
+            'toot'        => 'boolean',
+            'ibnr'        => 'boolean',
+            'tripID'      => 'required',
+            'lineName'    => 'required',
```

Comment:
Can you define whether all the things are `nullable` or `required`? Just to have a good overview.